README.md

```diff
@@ -1,24 +1,31 @@
-[![Join the chat at https://gitter.im/zlsa/atc](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zlsa/atc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
@erikquinn I *love* the first set.  

I'd be ok with adding the coverage badge, but I'd like to get our coverage up a bit before that happens. 😬